chess/ChessGameModel.swift

```diff
@@ -137,30 +138,40 @@ extension ChessGameModel {
     }
 
     private func handle(_ message: String) {
-        guard isValidNotation(message) || isValidColor(message) else {
-            self.logger.error("Received invalid message: \(message)")
-            self.gameStatus = "Invalid move"
-            self.board = previousBoardState
-            self.player?.isMyTurn.toggle()
+        let messageType = findTypeOf(message)
```

Comment:
this method sounds "unswifty" 😄  Usually finding type is about swift type. Alternatively you can write an init for MessageType enum

Reply:
Made the init